### ISSUE TITLE
Installing Flarum

### ISSUE THREAD

**Issue by alexbagirov** [flarum/core#58]
Hello. As I understand I need to use virtual machines and other virtualization stuff to use the forum. 
Is it possible to install it as esoTalk by uploading to the hosting and just going to installation page?

**Comment by thangngoc89** [comment]
Hello,
This question has been asked for many times. The virtualbox is developement environment. You can install Flarum like any other PHP script when these is a release

**Comment by alexbagirov** [comment]
So it is not possible right now? (I mean installing as all PHP scripts)

**Comment by Holyphoenix** [comment]
It would be possible to get it up and running on shared hosting or vps hosting without the visualization, but you would have to set up all of the backend stuff manually and the Flarum team will not really help you to do so(since it's for developers only right now). There is no installation script yet, that will be created once Flarum is ready to be used on production websites.

**Comment by alexbagirov** [comment]
Thank you!
Don't you know when it is going to have an initial release?

**Comment by Holyphoenix** [comment]
No idea.  I know Toby and Franz have been very hard at work on it.  I'd expect an initial release early next year.  But that is a guess.  

If you start your forum with Esotalk or FluxBB, there will be a migration script to Flarum almost immediately after it's release.  So if you need something now, and are already sold on Flarum, I'd recommend using one of those two scripts for the time being.

**Comment by alexbagirov** [comment]
Oh, so much to wait :(

**Comment by tobscure** [comment]
Aiming for an early, installable release in a few months time.

**Comment by franzliedke** [comment]
@alexbagirov Check out [this topic](http://discuss.flarum.org/116-model-class-in-a-core-repositary-of-flarum-is-conflict-with-lar) on our development forum. It is possible to set up Flarum without virtual machines etc.

**Comment by alexbagirov** [comment]
Ok, will try to go through these steps. 
@franzliedke, do I have to install Composer to do that or all files for that are in the repository already?

**Comment by alexbagirov** [comment]
While going through these steps I get an error (step 3) with `composer install --prefer-dist` inside flarum folder. I have this.
![2015-04-17 18 08 58](https://cloud.githubusercontent.com/assets/6885864/7202548/de5abbb8-e52c-11e4-954e-6fee4aea2ddc.png)

**Comment by franzliedke** [comment]
Let's discuss this on the forums, okay?

**Comment by alexbagirov** [comment]
Ok, thanks!
But look for my another issue, please. I can't use forum because of that.
